cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestWaterContractTypeController.java

```diff
@@ -0,0 +1,40 @@
+package mil.army.usace.hec.cwms.radar.client.controllers;
+
+import mil.army.usace.hec.cwms.radar.client.controllers.WaterContractTypeEndpointInput.GetAll;
+import mil.army.usace.hec.cwms.radar.client.model.RadarObjectMapper;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+import java.util.List;
+
+import mil.army.usace.hec.cwms.radar.client.model.LookupType;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TestWaterContractTypeController extends TestController {
+    @Test
+    void testRetrieveAll() throws IOException {
+        String collect = readJsonFile("radar/v1/json/water_contract_types.json");
+        mockHttpServer.enqueue(collect);
+        mockHttpServer.start();
+        GetAll input = WaterContractTypeEndpointInput.getAll("SPK");
+        List<LookupType> values = new WaterContractTypeController()
```

Comment:
If we're doing a getAll, we should make sure we get multiple values, and validate multiple values.

Reply:
Added validation for other values returned by getAll